inactive_issues/inactive_issues_scraper.py

```diff
@@ -47,7 +47,7 @@ def run(issues):
 def inactive_issues_json(request):
     try:
         GH_TOKEN = get_api_key('GH')
-    except:
+    except Exception:
```

Comment:
Separate commit? :)

Reply:
I normally am happy to include minor fixes in the 'upgrade coafile' patch, so we can see what was problematic at the time.

Reply:
Ah I forgot that Pycodestyle gets now enabled because the typo is fixed... Nvm :+1: